MonMaraicher/Features/Map/MapView.swift

```diff
@@ -35,9 +35,29 @@ struct MapView: View {
             MapUserLocationButton()
         }
         .onAppear(perform: viewModel.onViewAppear)
+        .overlay(alignment: .bottomTrailing) {
+            nearbyFarmerButton
+        }
     }
 }
 
 #Preview {
```

Comment:
Try to organize your file declarations in the order of the most important first. So to understand your view, your extension is more important than your Preview. So I would put the preview at the end.